### DISCUSSION TITLE
unable to login

### DISCUSSION THREAD

**Discussion by metoo2008** [labring/FastGPT#957] [General]
wsl环境下安装一切正常，但启动打开后无法登录
![fastgpt无法登录](https://github.com/labring/FastGPT/assets/16477845/b6b4d97d-6ec1-4ae3-9d65-7179cf05504f)

**Comment by qinglinxin-195328** [comment]
我也遇到同样的问题，求解答

**Comment by diyeryang** [comment]
我在云上部署同样是这个问题，求解答

**Comment by geminizyz** [comment]
doker logs mongo 看看mongo的运行状态  ；我遇到同样的前台报错是 mongodb没有初始化完成；我得原因是 我 docker pull了 mongo 最新版，然后 按照 文档 要求的 版本 重新部署mongodb就好了
